src/main/kotlin/net/atos/zac/admin/model/ReferenceTable.kt

```diff
@@ -78,3 +80,12 @@ fun ReferenceTable.toRestReferenceTable(inclusiefWaarden: Boolean): RestReferenc
         restReferenceTableValues
     )
 }
+
+fun ReferenceTable.updateExistingReferenceTable(
```

Comment:
Should the `ReferenceTable` be here? Or maybe the question should be: should the entity and rest model classes be in the same package?

Reply:
O my bad - they are not in the same package - but next question then:

I would not expect this method to exist here, as it creates a dependency for the ReferenceTable file on the `app.admin.model` package. Smells like package tangle !

Reply:
yeah, I agree. I will move it!